specs-go/config.go

```diff
@@ -615,6 +615,19 @@ type Arch string
 // LinuxSeccompFlag is a flag to pass to seccomp(2).
 type LinuxSeccompFlag string
 
+const (
+	// LinuxSeccompFlagLog is a seccomp flag to request all returned
+	// actions except SECCOMP_RET_ALLOW to be logged. An administrator may
+	// override this filter flag by preventing specific actions from being
+	// logged via the /proc/sys/kernel/seccomp/actions_logged file. (since
+	// Linux 4.14)
+	LinuxSeccompFlagLog LinuxSeccompFlag = "SECCOMP_FILTER_FLAG_LOG"
```

Comment:
@thaJeztah since the name of these variables is different than in [your PR](https://github.com/opencontainers/runtime-spec/pull/1108/files#diff-9790b3b8b39fc583eab8f6d97ac27ae3f35dbe339000c746faf64615125a8ec7R619) (`FlagLog` and `FlagAllow`) I'm wondering if Docker/moby has already _used_ your variables?
I'm inclined to merge this PR given the robust comments and that it drops the TS_SYNC flag that @rata called out